c10/core/TensorImpl.h

```diff
@@ -598,6 +600,24 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
    */
   const at::Tensor& grad() const;
 
+  /**
+   * Return the accumulated gradient of a tensor. This gradient is computed
+   * using forward mode AD.
+   *
+   * It is only valid to call this method on a Variable.
+   * See Note [Tensor versus Variable in C++].
+   */
+  const at::Tensor& fw_grad(uint64_t level, const at::Tensor& self) const;
+
+  /**
+   * Sets the forward gradient for this Tensor.
+   * The given Tensor might not be used directly and its content will be copied.
+   *
+   * It is only valid to call this method on a Variable.
+   * See Note [Tensor versus Variable in C++].
```

Comment:
This Note doesn't exist anymore

Reply:
I re-used the comment in other functions in this file. I didn't knew this has been removed.
Are all these functions valid to call on any Tensor now and these two lines can be removed for all of the functions here?

Reply:
I re-used the comment for other functions in this file to write this one. I didn't knew this has been removed.
Are all these functions valid to call on any Tensor now and these two lines can be removed for all of the functions here?

Reply:
Yes, I think the note is removed because it was obsolete, so you can remove it everywhere now.

Reply:
Ok I'll remove it from these and do another PR to remove it from all other functions in this file.